sql/catalyst/src/main/java/org/apache/spark/sql/connector/expressions/Expression.java

```diff
@@ -17,7 +17,7 @@
 
 package org.apache.spark.sql.connector.expressions;
 
-import java.util.Arrays;
+import java.util.*;
```

Comment:
Nit: I think we've avoided wildcard imports, just enumerate them?

Reply:
done